kubernetes/zenko/charts/burry/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for elasticsearch-curator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  # At 05:00 every day
+  schedule: "0 5 * * *"
+  annotations: {}
+  concurrencyPolicy: Forbid
```

Comment:
Things that should be configurable, should be exposed as values. Does anything other than `concurrencyPolicy: Forbid` make sense for this use-case?